design/pedibus/override/templates/not_avaiability/not_avaiability.tpl

```diff
@@ -0,0 +1,220 @@
+{set-block scope=root variable=cache_ttl}0{/set-block}
+{def $__OBJECTS = fetch(
```

Comment:
fetch di lista senza limit: pericolosa per le perfomance

Reply:
Qui secondo me non si può limitare ne per valore di limit, ne per data.
Il fetch prende sia oggetti di classe **Sostituzione volontario** che di classe **Assenza volontario** di cui solo **Assenza volontario** potrebbe essere filtrabile per data. Se inserisco un limit di 180 giorni, ovvero dei giorni scolastici, se non vengono eliminate le vecchie assenze e sostituzione, l'anno scolastico successivo non visualizzerà più tutte le assenze e sostituzioni correnti.

Reply:
Per una questione di storico, gli utenti devono poter vedere il calendario di tutto l'anno scolastico(quindi date anche passate) per poter vedere le assenze e sostituzioni accadute.
Si potrebbe fare due fetch distinte, per le due tipologie di oggetti, previa cambiamento datatype **relazioni oggetti** (da voi impostato) nella classe **Sostituzione volontario** con una **relazione oggetti** in modo da poter filtrare per anno scolastico (Esempio dal 01.09.2017 al 01.07.2018) e quindi non prendere le date dell' anno scolastico precedente; 
Sarebbe possibile farlo perché per ora non sono state inserite sostituzioni.